krux_boto/util.py

```diff
@@ -8,6 +8,7 @@
 #
 
 from __future__ import absolute_import
+import string
```

Comment:
This looks incorrect. a) it's unusual to import string anymore and b) I don't see you adding usage of it anywhere.